src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.Reflection.cs

```diff
@@ -111,6 +111,12 @@ private static bool IsType([NotNullWhen(true)] INamedTypeSymbol? namedTypeSymbol
 		{
 			if (namedTypeSymbol != null)
 			{
+				if (typeSymbol?.SpecialType == SpecialType.System_Object)
+				{
+					// Everything is an object.
+					return true;
+				}
```

Comment:
Previously, the following error was produced:

```
Cannot assign child of type 'Windows.UI.Xaml.Controls.ToolTip' to property of type 'object'.'
```

This is because the below loop stops at System_Object, so if you ask `IsType(anyNamedType, objectType)` the method will return false incorrectly.

There are two possible solutions I'm seeing.

1. The solution currently done in the PR to return true directly for System.Object
2. Change the condition of the below loop to be `namedTypeSymbol != null` instead of `namedTypeSymbol.SpecialType != SpecialType.System_Object`

I don't have a strong opinion. @jeromelaban thoughts?

Reply:
I don't have strong opinion on this one either. If we're assigning anything to an object, it should always be valid, so I'd say the fastest determination technique should be used.

Reply:
@jeromelaban Ok, let's keep it with the condition for now. We can always revisit.